core/src/rpc.rs

```diff
@@ -796,6 +795,9 @@ pub trait RpcSol {
         start_slot: Slot,
         end_slot: Slot,
     ) -> Result<Vec<String>>;
+
+    #[rpc(meta, name = "getLowestNonpurgedBlock")]
```

Comment:
Needs a doc/ update of course

Reply:
`getLowestAvailableBlock` to avoid the double negative?  `getFirstAvailableBlock`?

Reply:
I like ` getFirstAvailableBlock`; do you think it's clear enough that it's the first available after ledger cleaning (as opposed to some other gap)? I suppose docs will help with that 😆

Reply:
Yeah I like that.